report/src/introduction/waterfall_vs_agile.tex

```diff
@@ -1,2 +1,60 @@
+
+The process for working as a Scrum team boils down to repeatedly taking backlog items and working on them during a sprint. After the sprint, the team and stakeholders inspect the results and adjust for the next sprint. This process is what fosters continuous improvement by getting rapid feedback on work.
+
+During this report, we will present the work done during sprints. This includes the initial planning, and later the product feedback received.
+To find common understanding of the process, we present definitions of the artifacts and tools used during the Scrum process below.
+
+One of the first things a Scrum team implements is the \textbf{product backlog}. This is a list of user stories, and represents every possible addition to the project. Due to the comprehensive nature of this list, it is not possible to build everything. Therefore, the list should be prioritized by what delivers the most value to users.
+Backlog items then are estimated by the team in terms of relative size. A common measure for estimation is the Fibonacci sequence.
+As the stakeholders provide feedback, items in the backlog should be prioritized accordingly. All these items contribute to achieving the \textbf{product goal}.
+
+The backlog items are \textbf{user stories}. These stories represent functionality the stakeholder would like to see in the system. For this reason, they must include \textit{who} it is for, \textit{what} should be done, and \textit{why} they want it. This is captured in the general "\texttt{As a X, I want Y, such that Z.}" format for user stories.
+
+A \textbf{sprint} is a short period of time, during which the team works on a subset of the product backlog, captured in a \textbf{sprint backlog}. This defines the \textit{what} of the sprint. The items in the sprint backlog are chosen based on a \textbf{sprint goal}, defining the \textit{why} of the sprint. The sprint also has an actionable plan for delivering the increment, representing the \textit{how}.
+
+Sprints start by planning them. During \textbf{sprint planning}, the team discuss and choose the most important backlog items. Then they find a sprint goal which clarifies why the sprint is valuable to the users. The team also discussed how the chosen work will get done in increments that meet the definition of done. \textbf{Increments} can be viewed as stepping stones toward the product goal. Each one is better than the previous, as measured in terms of progress towards the goal. It is possible to have multiple increments during a spring. Increments have requirements for when they are considered done. This is the \textbf{definition of done}. Usually, simply implementing a feature does not meet this definition. Testing, ensuring deployability, and writing documentation may also be required.
```

Comment:
Meget lang section. Der mangler kilder for DoD. DoD kunne meget godt komme foer.

Reply:
Reworded